bin/ungit

```diff
@@ -21,19 +39,15 @@ process.on('uncaughtException', function(err) {
   });
 });
 
-var child = new (forever.Monitor)(path.join(__dirname, '..', 'src', 'server.js'), {
-  silent: false,
-  minUptime: 2000,
-  max: config.maxNAutoRestartOnCrash,
-  cwd: path.join(process.cwd(), '..'),
-  options: process.argv.slice(2),
-  env: { LANG: 'en_US.UTF-8' }
-});
-
 child.on('exit', function (res) {
   console.log('Stopped keeping ungit alive');
 });
 
+process.on('SIGTERM', function() {
```

Comment:
Is this still needed?
You are currently handling the `SIGTERM` event in two places.